phpBB/posting.php

```diff
@@ -1388,8 +1386,6 @@
 				'post_author_name',
 				'update_message',
 				'update_subject',
-				'submit',
-				'error',
```

Comment:
in case of before, I'd prefer to fix error.
So an extension can set it and the submit is aborted.